src/dmd/inline.d

```diff
@@ -226,25 +216,54 @@ public:
     }
 
     override void visit(IfStatement s)
+    {
+        visitIfStatement(s, null);
+    }
+
+    extern (D) private void visitIfStatement(IfStatement s, void delegate(DoInlineAs!Result) visitSubseq)
```

Comment:
Does this really need to be declared `extern(D)`?

Reply:
Yes, because C++ does not have delegates.